lib/tasks/argo.rake

```diff
@@ -279,29 +279,6 @@ namespace :argo do
     puts "#{facet.items.count} Workgroups:\n#{facet.items.collect(&:value).join(%[\n])}"
   end
 
-  desc "Update the .htaccess file from indexed APOs"
-  task :htaccess => :environment do
-    directives = ['AuthType WebAuth',
-                  'Require privgroup dlss:argo-access',
-                  'WebAuthLdapAttribute suAffiliation',
-                  'WebAuthLdapAttribute displayName',
-                  'WebAuthLdapAttribute mail']
-
-    directives += (File.readlines(File.join(Rails.root, 'config/default_htaccess_directives')) || [])
-    facet = get_workgroups_facet()
-    unless facet.nil?
-      facets = facet.items.collect &:value
-      priv_groups = facets.select { |v| v =~ /^workgroup:/ }
-      directives += priv_groups.collect { |v|
-        ["Require privgroup #{v.split(/:/,2).last}", "WebAuthLdapPrivgroup #{v.split(/:/,2).last}"]
-      }.flatten
-      File.open(File.join(Rails.root, 'public/.htaccess'),'w') do |htaccess|
-        htaccess.puts directives.sort.join("\n")
-      end
-      File.unlink('public/auth/.htaccess') if File.exist?('public/auth/.htaccess')
```

Comment:
@eefahy pointed out that it's unnecessarily circuitous.  this code queries the existing objects for permissions related info, and builds up a list from the permissions that are currently in use.  but that's unnecessary, since we know all the workgroups we'll care about supporting.  and now that i think about it, we might want to add them to the htaccess file before they're listed in an APO.  if we define new workgroups, which we do occasionally, we can manually add them to whatever config puppet uses.  and as per the commit message, updating htaccess doesn't really correspond with deploying/restarting the app, which is when this rake task is run.

it's clever, but it doesn't really gain us anything, as far as we can tell.